.github/workflows/ci.yaml

```
@@ -287,3 +287,47 @@ jobs:
             scripts/images/**/flatcar_production_*.sh
             scripts/images/**/flatcar_test_update.gz
             scripts/ebuild_logs.tar.xz
+  test:
+    name: "Run kola tests"
+    needs: packages
+    runs-on:
+      - debian
+      - build
+      - self-hosted
+      - x64
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: ["amd64", "arm64"]
+    defaults:
+      run:
+        working-directory: scripts
+
+    steps:
+      # it downloads from the previous linked job.
+      - name: Download artifacts
+        uses: actions/upload-artifact@v3
```

This should probably be ```actions/download-artifact@v3```
```suggestion
        uses: actions/download-artifact@v3
```